src/travis2docker/templates/Dockerfile

```diff
@@ -58,9 +58,9 @@ WORKDIR ${TRAVIS_BUILD_DIR}
 
 {% if runs -%}
 {% if image == 'quay.io/travisci/travis-python' -%}
-RUN /bin/bash -c "source $HOME/virtualenv/python2.7_with_system_site_packages/bin/activate && {{ ' && '.join(runs) }}"
+RUN /bin/bash -c "source $HOME/virtualenv/python2.7_with_system_site_packages/bin/activate && source /home/travis/.rvm/scripts/rvm  && {{ ' && '.join(runs) }}"
 {% else %}
-RUN /bin/bash -c "source /usr/local/rvm/scripts/rvm && {{ ' && '.join(runs) }}"
+RUN /bin/bash -c "test -f /usr/local/rvm/scripts/rvm && source /usr/local/rvm/scripts/rvm; {{ ' && '.join(runs) }}"
```

Comment:
This won't work if RVM is installed under the user on the unknown image. RVM recommends that we do the following, which covers the Travis, the known, and the unknown:

```
\# Load RVM into a shell session *as a function*
if [[ -s "$HOME/.rvm/scripts/rvm" ]] ; then

  \# First try to load from a user install
  source "$HOME/.rvm/scripts/rvm"

elif [[ -s "/usr/local/rvm/scripts/rvm" ]] ; then

  \# Then try to load from a root install
  source "/usr/local/rvm/scripts/rvm"

else

  printf "ERROR: An RVM installation was not found.\n"

fi
```